src/track/text-style.ts

```diff
@@ -181,19 +189,19 @@ class TextStyle {
     return textStyle;
   }
 
-  public static toJson(text: TextStyle): PKTextStyleObject {
-    return {
-      fontEdge: text.fontEdge,
-      fontSize: text.fontSize,
-      textAlign: text.textAlign,
-      fontScale: text.fontScale,
-      fontColor: text.fontColor,
-      fontOpacity: text.fontOpacity,
-      backgroundColor: text.backgroundColor,
-      backgroundOpacity: text.backgroundOpacity,
-      fontFamily: text.fontFamily
-    };
-  }
+public static toJson(text: TextStyle): PKTextStyleObject {
+  return {
```

Comment:
note the lint errors
looks like lint and prettier are conflicting on this file
you can do "save without formatting" to save without applying prettier